packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -206,11 +209,13 @@ class Paginator extends React.Component {
     const fullView = ariaLabelledBy
       ? (
         <nav className={cx('paginator', !totalCount && 'pageless', theme.className)} aria-labelledby={ariaLabelledBy}>
+          <VisuallyHiddenText aria-live="polite" aria-relevant="all" text={this.state.selectedPageMessage} />
```

Comment:
nit: can we move this to a const like `fullViewChildren`?

Same for the other paginators as well

Reply:
https://github.com/cerner/terra-core/pull/3886/commits/f0af8740cfd4e4ebb8736772976eb7dba772ad57#diff-0bceee42b89777a164b9a100804131b9447d947169e02cffe25e8791c74a958aR209-R212